packages/frontend/app/(dashboard)/organization/settings/page.tsx

```diff
@@ -30,7 +30,26 @@ import { useOrganizationSettings } from '@/app/hooks/useOrganizationSettings'
 import { checkCourseCreatePermissions } from '@/app/utils/generalUtils'
 
 export default function SettingsPage(): ReactElement {
+  // Handler to update SSO patterns
+  const updateSSO = async () => {
+    try {
+      const formValues = await formSSO.validateFields()
+      const ssoEmailPatterns = formValues.ssoEmailPatterns || []
+      await API.organizations.patch(organization?.id ?? -1, {
+        ssoEmailPatterns,
+      })
+      message.success('SSO patterns updated')
+      setTimeout(() => {
+        window.location.reload()
+      }, 1750)
+    } catch (error: any) {
+      const errorMessage =
+        error?.response?.data?.message || 'Failed to update SSO patterns'
+      message.error(errorMessage)
```

Comment:
```suggestion
      const errorMessage = getErrorMessage(error)
      message.error(errorMessage)
```
Just so you know for the future, we have a function called `getErrorMesssage`  so that way we have like a central way of retrieving the error messages we get from out endpoints (because the error messages aren't always in the same object format each time for some reason)